File: ERPIN.Domain/Entities/INV/Item.cs
using ERPIN.Domain.BaseModels;
using ERPIN.Domain.Enums;
using System.ComponentModel.DataAnnotations.Schema;

namespace ERPIN.Domain.Entities.INV;

[Table("Items", Schema = nameof(Schemes.INV))]
public class Item : BaseNamedEntity
{
    public required string BarCode { get; set; }
    public string? Description { get; set; }
    public string? UOM { get; set; }
    public decimal? PurchasePrice { get; set; }
    public decimal? SellPrice { get; set; }
    public bool IsActive { get; set; }

}

File: ERPIN.Presentaion/src/app/Interfaces/Response/Item.ts
import { BaseNamedEntity } from './BaseNamedEntity';

export interface Item extends BaseNamedEntity {
  barCode: string | null;
  description: string | null;
  uOM: string | null;
  purchasePrice: number | null;
  sellPrice: number | null;
  isActive: boolean;
}
